composer/modules/web/webpack.config.js

```diff
@@ -213,6 +235,7 @@ const config = [{
             event_channel: 'core/event/channel',
             plugins: 'plugins',
             images: 'public/images',
+            '../../theme.config$': path.join(__dirname, 'src/ballerina-theme/theme.config')
```

Comment:
This is a bit weird name for an alias. Why ../.. ? Wouldn’t just theme.config$ be less confusing?

Reply:
This is because `semantic-ui-less was` not built to be easily extendable it always searches for the theme.config file within its root package directory using a relative path. so we are mapping `../../theme.config$` to our theme directory.